### DISCUSSION TITLE
Both statusbars

### DISCUSSION THREAD

**Discussion by WS64** [koreader/koreader#7176] [Q&A] [answered]
Is it possible to have both status bars?
E.g. to show title and author on top and the rest on bottom...
I cannot find a way to activate both.

**Comment by poire-z** [comment, marked as answer]
It is possible: when enabling Alt status bar, it disables footer - but you can still re-enable it by tapping on it (if you haven't checked Lock status bar in the footer menu). Or something like that.

**Comment by WS64** [comment]
Thanks!
Since I had it locked I never knew it could be turned on again.
But why does the alter disables the normal first?
Seems more confusing to me than helping!

**Comment by WS64** [comment]
Btw, there seems to be a bug in turing the normal status bar on again.
It seems not to work at first until I close and reopen the book.

**Comment by NiLuJe** [comment]
On which version?

**Reply by WS64** [reply]
2020.12-124

**Reply by NiLuJe** [reply]
Hmm, yeah, opening a book while having the Footer disabled disables its event handlers (`disableFooter`), which probably prevents it from being enabled back until the next open.

**Reply by NiLuJe** [reply]
A simpler workaround would probably be simply not to hide the Footer when enabling the Header. (That's `ReaderRolling:onSetStatusLine`).

**Reply by NiLuJe** [reply]
(Would allow one to get rid of the extra arg workaround this event can take).

**Reply by WS64** [reply]
agreed

**Reply by ghost** [reply]
I would prefer that too (not hiding the footer). 
But don't we have a problem with the name "alternate status bar" as alternate sounds like mutual exclusive the one or the other.
Would it be possible to name them header/footer top/bottom or similar?

**Reply by WS64** [reply]
> Would it be possible to name them header/footer top/bottom or similar?
Agreed, "header bar" and "footer bar"

**Reply by Frenzie** [reply]
> But don't we have a problem with the name "alternate status bar" as alternate sounds like mutual exclusive the one or the other.

It's mutually exclusive with non-cre documents, so while I'm not quite sure if it indeed suggests that, I'd consider that an argument *in favor*. ;-)

> Would it be possible to name them header/footer top/bottom or similar?

If by "similar" you mean something like "main/alternate", sure. Otherwise strongly opposed.

**Comment by ghost** [comment]
Hmmm, why did I know that before? ;)

Am 22. Jänner 2021 21:18:17 MEZ schrieb Frans de Jonge <notifications@github.com>:
>> But don't we have a problem with the name "alternate status bar" as
>alternate sounds like mutual exclusive the one or the other.
>
>It's mutually exclusive with non-cre documents, so while I'm not quite
>sure if it indeed suggests that, I'd consider that an argument *in
>favor*. ;-)
>
>> Would it be possible to name them header/footer top/bottom or
>similar?
>
>If by "similar" you mean something like "main/alternate", sure.
>Otherwise strongly opposed.
>
>-- 
>You are receiving this because you commented.
>Reply to this email directly or view it on GitHub:
>https://github.com/koreader/koreader/discussions/7176#discussioncomment-303413

-- 
Diese Nachricht wurde von meinem Android-Gerät mit K-9 Mail gesendet.
